phalcon/Forms/Form.zep

```diff
@@ -598,7 +617,7 @@ class Form extends Injectable implements Countable, Iterator, AttributesInterfac
      * @param array data
      * @param object entity
      */
-    public function isValid(var data = null, var entity = null) -> bool
+    public function isValid(var data = null, var entity = null, var whitelist = null) -> bool
```

Comment:
Why not make `whitelist` always `array` with default value `[]`?

Reply:
Also, please update `@param` in docblock.

Reply:
`whitelist` needs to be an array here. One less check in the `if`.